src/pages/sns/faq.tsx

```diff
@@ -689,6 +698,64 @@ function SnsFaqPage() {
                 </Link>
               </ul>
             </Faq>
+          </FaqSection>
+
+          <FaqSection
+            id="treasury"
+            title={
+              <h2 className="tw-heading-3 text-gradient mb-12 md:mb-0 md:tw-heading-60">
+                SNS DAO treasury
+              </h2>
+            }
+          >
+            <Faq title="What is the SNS treasury?">
+              After a decentralization swap successfully concludes, the ICP acquired through the
```

Comment:
```suggestion
              After a decentralization swap successfully concludes, the ICP collected through the
```
(we often use "collect funds" but I think acquireing also works)